services/painting-surface-data-simulator-service/app/services/model_client.py

```
@@ -0,0 +1,173 @@
+    async def health_check(self) -> bool:
+        """도장 표면 결함탐지 서비스 헬스 체크"""
+        if not self.service_url:
+            return False
+
+        health_url = f"{self.service_url}/health"
+
+        try:
+            async with httpx.AsyncClient(timeout=self.timeout) as client:
+                response = await client.get(health_url)
+                return response.status_code == 200
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)